### ISSUE TITLE
Trigger chars not fully replaced in LibreOffice Writer

### ISSUE THREAD

**Issue by jbrooksaz** [federico-terzi/espanso#1992]
### Description

Running LibreOffice v 7.3.7.2 on fully updated Pop!_OS 22.04 LTS.  Espanso v2.2.1, installed with the .deb. Messed around with inject_delay and backspace_delay so that Firefox and *all* other apps work perfectly with Espanso, *except* LibreOffice Writer.  At least it's consistent: the first character of the trigger remains while the rest of the replacement works.  Example: 
trigger: ";phx"
replace: "phoenix"
Results in:  ;hoenix
I tried creating a writer.yml per other people's attempts to fix this. Tried option backend: Clipboard, then tried force_mode: clipboard, with no change in Writer behavior at all.  All the other LibreOffice apps work great with Espanso.

### Steps to reproduce

1. Create a trigger/replace entry in base.yml. 
2. Execute the trigger in LibreOffice Writer.
The result will be the first character of the trigger is not replaced by the first character of the replacement string. The rest of the replacement string works correctly.

### Expected behavior

The first char of the trigger should be replaced by the first char of the replacement string.

### Your environment

- LibreOffice:  Version: 7.3.7.2 / LibreOffice Community
Build ID: 30(Build:2)
CPU threads: 12; OS: Linux 6.9; UI render: default; VCL: gtk3
Locale: en-US (en_US.UTF-8); UI: en-US
Ubuntu package version: 1:7.3.7-0ubuntu0.22.04.5
Calc: threaded
- uname: Linux sunrise 6.9.3-76060903-generic #202405300957~1718348209~22.04~7817b67 SMP PREEMPT_DYNAMIC Mon J x86_64 x86_64 x86_64 GNU/Linux
- Pop!_OS 22.04 LTS; Gnome shell 42.9

### Self-service

- [ ] I'd be willing to fix this bug myself.

**Comment by smeech** [comment]
Not sure if I can help with this - I get occasional instances of this problem in various programs, but not Writer today!

Would you like to post the output of `#acfg#` in case anyone had any suggestions?

You might like to post the problem on the Discord server, where it'll probably get more attention.

**Comment by jbrooksaz** [comment]
Thanks for your suggestion, I think I may have tripped across the problem. Maybe. I found that espanso works perfectly with Writer on my System76 laptop, running the exact same OS etc.  The difference was in the installation. On my laptop, I installed using AppImage because the .deb had errors; on my desktop, I got lazy and installed using the .deb and ignored the errors. Espanso still ran, I didn't think much of it, but now I think I'm paying the price... Going to uninstall the .deb and use AppImage and see if that solves the problem when I can find enough minutes in an hour... Will update the results.

**Comment by jbrooksaz** [comment]
Think I figured it out, posting so others who are having same issue might try this. Just to be thorough, I rebuilt my desktop machine from the ground up (System76 machine with Pop_OS) since I'd run AutoKey on it prior and didn't want any left-overs that might conflict. Ended up still having same problem - Espanso working perfectly on all my apps except LibreOffice Writer.  Then I did a #acfg# dump and compared my System76 laptop vs desktop. Found 2 config settings were different: the backend var set to 'Auto' fails, and set to 'Clipboard' works; the second var is the key_delay, which was missing from the installed default.yml but present in the laptop. Added that set to 0 (zero).  Once I made those 2 changes, I was cooking with gas!  Works perfectly with Writer.  Thanks to smeech above.
